Core/Core/ar.lproj/Localizable.strings

```diff
@@ -169,24 +163,6 @@
 /* No comment provided by engineer. */
 "Attempt %d" = "المحاولة %d";
 
-/* No comment provided by engineer. */
-"Attribution" = "السمة";
-
-/* No comment provided by engineer. */
-"Attribution No Derivatives" = "سمة بدون مشتقات";
-
-/* No comment provided by engineer. */
-"Attribution Non-Commercial" = "سمة غير تجارية";
-
-/* No comment provided by engineer. */
-"Attribution Non-Commercial No Derivatives" = "سمة غير تجارية بدون مشتقات";
-
-/* No comment provided by engineer. */
-"Attribution Non-Commercial Share Alike" = "سمة غير تجارية بمشاركة المتشابهين";
-
-/* No comment provided by engineer. */
-"Attribution Share Alike" = "سمة المشاركة أيضًا";
-
```

Comment:
These strings are still used & needed. I'm wondering if our NSLocalizedString func (with no bundle & comment) we defined isn't getting picked up right by this process, or if the xliff files we get back don't tell us where they belong anymore.